src/test/java/api/loans/scenarios/ChangeDueDateByReplacingLoanTests.java

```diff
@@ -339,4 +341,23 @@ public void dueDateCannotBeChangedWhenItemIsDeclaredLost() {
     assertThat("Should not contain check out service point summary",
       loanInStorage.containsKey("checkoutServicePoint"), is(false));
   }
+
+  @Test
+  public void cannotChangeDueDateForAgedToLostItem() {
+    final DateTime newDueDate = DateTime.now(DateTimeZone.UTC).plusWeeks(4);
+
+    val agedToLostResult = ageToLostFixture.createAgedToLostLoan();
+
+    final JsonObject changedLoan = loansClient.getById(agedToLostResult.getLoanId())
+      .getJson().copy()
+      .put("action", "dueDateChange")
+      .put("dueDate", newDueDate.toString());
+
+    final Response replaceResponse = loansFixture.attemptToReplaceLoan(
+      agedToLostResult.getLoanId(), changedLoan);
+
+    assertThat(replaceResponse.getJson(), hasErrorWith(allOf(
+      hasMessage("item is Aged to lost"),
+      hasUUIDParameter("itemId", agedToLostResult.getItemId()))));
+  }
```

Comment:
Similarly, can these similar API tests be consolidated and replaced with unit tests, like in other recent pull requests?